### PR TITLE
Fixing discrepancies between entities and the db

### PR DESCRIPTION
**Pull request by Crutovel**
We have coordinator in Locations with ManyTo One relation, and its working.
Scripts and entities have been changed.

### CODE REVIEW

File: src/main/java/com/softserve/teamproject/entity/Location.java [hunk 1]
```diff
@@ -28,19 +28,21 @@
...
+  @JoinColumn(name = "coordinator_id", referencedColumnName = "id")
+  private User coordinator;
+
+  @ManyToOne
   @JoinColumn(name = "country_id", referencedColumnName = "id", nullable = false)
   private Country country;
 
-  @OneToOne
-  @JoinColumn(name = "coordinator_id", referencedColumnName = "id", nullable = false)
-  private User coordinator;
```
Comment (svoronin) [inline src/main/java/com/softserve/teamproject/entity/Location.java]: Is it okay to change this one-to-one to many-to-one?

File: src/main/java/com/softserve/teamproject/entity/Location.java [hunk 2]
```diff
@@ -28,19 +28,21 @@
...
-
   @OneToMany(cascade = CascadeType.ALL, mappedBy = "location")
   private Set<Group> groups;
 
   @OneToMany(cascade = CascadeType.ALL, mappedBy = "location")
   private Set<User> users;
 
+
+
```
Comment (svoronin) [inline src/main/java/com/softserve/teamproject/entity/Location.java]: Why did you add those two lines?

File: src/main/java/com/softserve/teamproject/entity/Location.java [hunk 3]
```diff
@@ -96,36 +97,14 @@ public void setUsers(Set<User> users) {
...
-  @Override
   public String toString() {
     return "Location{"
         + "id=" + id
         + ", name='" + name + '\''
         + ", country=" + country
-        + ", coordinator=" + coordinator
         + ", groups=" + groups
         + ", users=" + users
+        + ", coordinator=" + coordinator
```
Comment (svoronin) [inline src/main/java/com/softserve/teamproject/entity/Location.java]: Minor, but in general there is no need to commit shuffles like this.

File: src/main/java/com/softserve/teamproject/entity/Location.java (hunk 2 above)
Comment (Crutovel) [inline src/main/java/com/softserve/teamproject/entity/Location.java]: my bad

File: src/main/java/com/softserve/teamproject/entity/Location.java (hunk 1 above)
Comment (Crutovel) [inline src/main/java/com/softserve/teamproject/entity/Location.java]: I dont know for sure, but i can change if it nessesary .  
I thought that could be 2 coordinators in one location so thats why.

File: src/main/java/com/softserve/teamproject/entity/Location.java (hunk 3 above)
Comment (Crutovel) [inline src/main/java/com/softserve/teamproject/entity/Location.java]: okay i agree

File: src/main/java/com/softserve/teamproject/entity/Location.java (hunk 1 above)
Comment (svoronin) [inline src/main/java/com/softserve/teamproject/entity/Location.java]: I'd prefer not to change it right now. Please check if One-To-One works, and minimize changes if that schema works.

Comment (svoronin) [inline src/main/java/com/softserve/teamproject/entity/Location.java]: Try to keep the order of fields as they used to be

Comment (Crutovel) [inline src/main/java/com/softserve/teamproject/entity/Location.java]: okay

File: src/main/java/com/softserve/teamproject/config/SpringSecurityInit.java [hunk 4]
```diff
@@ -2,10 +2,6 @@
 
 import org.springframework.security.web.context.AbstractSecurityWebApplicationInitializer;
 
-/**
- * The class extends AbstractSecurityWebApplicationInitializer class that helps us map the
- * springSecurityFilterChain.
- */
```
Comment (thedikiy) [inline src/main/java/com/softserve/teamproject/config/SpringSecurityInit.java]: Why removing Javadoc?

Comment (Crutovel) [inline src/main/java/com/softserve/teamproject/config/SpringSecurityInit.java]: sry i thought that was unnecessary copy paste comment
